src/components/Edit/ChangeSummary.vue

```diff
@@ -26,17 +26,26 @@
         </v-row>
       </div>
     </template>
+
+    <template>
```

Comment:
You don't need the template. Above it was used a nice way to wrap both the divider and div in a v-if.

I recommend a block comment as well (useful when you use code folding to see the big picture).

Reply:
Sorry, missed my conditional here on the template. 
Will include a divider as well. 
Update inc

Reply:
Updated